master/buildbot/test/unit/test_scripts_cleanupdb.py

```diff
@@ -43,6 +43,19 @@ def mkconfig(**kwargs):
     return config
 
 
+def patch_environ(case, key, value):
```

Comment:
this function is generic, but its both usages are very specific: the same environment variable name, the same value. does it really have to be that generic?